pupil-spa/test/features/step_definitions/pupil_auth_steps.rb

```diff
@@ -0,0 +1,38 @@
+Then(/^I should see meta data stored in the DB$/) do
+  current_time = Time.now
+  wait_until {page.evaluate_script('window.localStorage.getItem("pupil");')}
+  storage_pupil = JSON.parse page.evaluate_script('window.localStorage.getItem("pupil");')
+  stored_check = MongoDbHelper.get_pupil_check_metadata(storage_pupil['checkCode'])
+  expect(stored_check['updatedAt'].strftime('%d-%m-%y %H:%M')).to eql current_time.utc.strftime('%d-%m-%y %H:%M')
+  expect(stored_check['createdAt'].strftime('%d-%m-%y %H:%M')).to eql current_time.utc.strftime('%d-%m-%y %H:%M')
+  expect(stored_check['pupilId']).to eql @pupil_information['_id']
+  check_window_id = stored_check['checkWindowId'].to_s
+  MongoDbHelper.get_check_window(check_window_id)
+  expect(MongoDbHelper.get_check_window(check_window_id)['endDate'] > current_time).to be_truthy
```

Comment:
Do you want to check that the start date is in the past too?

Reply:
Done!